sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1397,7 +1397,8 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif_t *netif,
         return false;
     }
 
-    if (IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_ARSM)) {
+    /* don't send multicast neighbor solicitation on 6lo */
+    if (!gnrc_netif_is_6lo(netif)) {
```

Comment:
Shouldn't ARSM still also be checked? It doesn't make sense to probe multicast, if there is no address resolution state machine.

```suggestion
 if (IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_ARSM) &&
     /* don't send multicast neighbor solicitation on 6lo */
     !gnrc_netif_is_6lo(netif)) {
```

Reply:
`_probe_nbr()` already gets [turned into a no-op](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.h#L243) when `CONFIG_GNRC_IPV6_NIB_ARSM` is not set, so we can remove the duplicate check and reduce clutter.

Reply:
It still would help ensuring the 6ln check is a no-op, but I get your point :-)